request_logging/middleware.py

```diff
@@ -90,6 +101,19 @@ def process_response(self, request, response):
 
         return response
 
+    def _log_multipart(self, body):
```

Comment:
Can you add a docstring explaining what this does? While you're in here, can you also modify `_chunked_to_max` to simply return `msg[:self.max_body_length]` without bothering to check the whole length for performance reasons? I know we lose the `...` indicating there was more data, but I think the tradeoff is worth it.

Reply:
Done